schedule/yam/agama_extensions_and_patterns_dev_group.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: agama_extensions_and_patterns_dev_group
```

Comment:
suggested in slack to change the ending to something more functional.

Reply:
Updated PR and MR, VRs updated also.